homeassistant/components/habitica/__init__.py

```diff
@@ -122,6 +143,73 @@ class HAHabitipyAsync(HabitipyAsync):
         def __call__(self, **kwargs):
             return super().__call__(websession, **kwargs)
 
+    async def cast_skill(call: ServiceCall) -> ServiceResponse:
+        """Skill action."""
+        entry: HabiticaConfigEntry | None
+        if entry := hass.config_entries.async_get_entry(call.data[ATTR_CONFIG_ENTRY]):
+            coordinator = entry.runtime_data
+            skill = {
+                "pickpocket": {"spellId": "pickPocket", "cost": "10 MP"},
+                "backstab": {"spellId": "backStab", "cost": "15 MP"},
+                "smash": {"spellId": "smash", "cost": "10 MP"},
+                "fireball": {"spellId": "fireball", "cost": "10 MP"},
+            }
+            try:
+                task_id = next(
+                    task["id"]
+                    for task in coordinator.data.tasks
+                    if call.data[ATTR_TASK] in (task["id"], task.get("alias"))
+                    or call.data[ATTR_TASK] == task["text"]
+                )
```

Comment:
What's a task?

Reply:
Oh lol, you execute these on tasks, interesting

So currently it can be either id, alias or text, sounds like a lot to support IMO, considered only using on of them?

Reply:
Yeah, you cast Brutal Smash on a Daily, To-do or Habit 💪🏼😁

I think ID is the best to identify a task, especially because title is potentially not unique. Being able to match by title would be consistent with the actions for to-dos. 

Alias is used in Habitica as an alias for the id  but it is something more advanced, as the input field is only available in the web interface if you activate developer tools, so maybe I could leave it out from the field description and only mention it in the documentation?

Reply:
How does alias work then? Isn't that the same as the issue text?

Reply:
nope, it is similar to a slug, only numbers, letters, `-` and `_` are allowed. In some API methods it can be used instead of the task id